examples/with-http2/next.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  // this needs to be set false unitl to a bug in the compression npm module gets fixed.
```

Comment:
```suggestion
  // this needs to be set false until a bug in the compression npm module gets fixed.
```
Might want to add a link in this comment to the corresponding issue on GitHub

Reply:
good point added it